roles/ceph-nfs/templates/ceph-nfs.service.j2

```diff
@@ -6,7 +6,7 @@ After=docker.service
 [Service]
 EnvironmentFile=-/etc/environment
 ExecStartPre=-/usr/bin/docker rm ceph-nfs-%i
-ExecStartPre=/usr/bin/mkdir -p /etc/ceph /etc/ganesha /var/lib/nfs/ganesha
+ExecStartPre=mkdir -p /etc/ceph /etc/ganesha /var/lib/nfs/ganesha
```

Comment:
Typically systemd is not happy if you don't pass full binary name, not sure why though.
Perhaps you could use jinja here to apply the right binary path, whether we run on Debian system or Red Hat system?

Reply:
Will fix this soon.